docs/extensions/httpfs.md

```diff
@@ -61,7 +61,15 @@ The `httpfs` filesystem is tested with [AWS S3](https://aws.amazon.com/s3/), [Mi
 
 ### Configuration
 
-To be able to read or write from S3, the correct region should be set:
+#### Automatic Settings and Credential Configuration
+
+The `httpfs` extensions automatically finds the local settings for the [AWS CLI client](https://aws.amazon.com/cli/), located in `~/.aws`, and uses its configuration (e.g., region) and credentials by default.
```

Comment:
Unless something has changed, this functionality is only in the AWS extension?

Reply:
@Mause that's correct!